customer-service/src/main/resources/application.properties

```diff
@@ -1 +1,6 @@
 server.port=8081
+spring.datasource.url=jdbc:postgresql://poss-db:5432/poss_db
+spring.datasource.username=poss
+spring.datasource.password=poss
```

Comment:
Fails [this](https://rules.sonarsource.com/secrets/RSPEC-6703/) rule.


__Suggestion:__

Use environment variables, with [`.env`](https://docs.docker.com/compose/environment-variables/env-file/) files.